pkg/recommendation/replicas/recommend.go

```diff
@@ -0,0 +1,318 @@
+package replicas
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"time"
+
+	predictionapi "github.com/gocrane/api/prediction/v1alpha1"
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/prediction/config"
+	"github.com/gocrane/crane/pkg/recommend/types"
+	"github.com/gocrane/crane/pkg/recommendation/framework"
+	"github.com/gocrane/crane/pkg/utils"
+	"github.com/montanaflynn/stats"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/yaml"
+)
+
+func (rr *ReplicasRecommender) PreRecommend(ctx *framework.RecommendationContext) error {
+	// we load algorithm config in this phase
+	// TODO(chrisydxie) support configuration
+	config := &config.Config{
```

Comment:
What if we have multiple algo config

Reply:
Can you describe the specific user case? Do we need hyper-parameter search?